Frontend/Views/Shared/_Layout.cshtml

```diff
@@ -96,7 +96,7 @@
                 <h2 class="govuk-heading-m">Get support</h2>
                 <div class="govuk-footer__meta-custom">
                     <ul class="govuk-footer__meta-custom govuk-list govuk-!-font-size-16">
-                        <li>Email: <a class="govuk-link govuk-footer__link" href="mailto:ServiceDelivery.RDD@education.gov.uk">ServiceDelivery.RDD@education.gov.uk</a></li>
+                        <li>Email: <a class="govuk-link govuk-footer__link" href="mailto:servicedelivery.rdd@education.gov.uk">ServiceDelivery.RDD@education.gov.uk</a></li>
```

Comment:
You've made the actual email link lowercase, great. But you also need to make the email text lowercase (by amending the anchor value after the href attribute) - let me know if that doesn't make sense.